src/machine/Coffee.java

```diff
@@ -0,0 +1,41 @@
+package machine;
+
+enum Coffee {
+    ESPRESSO(250, 0, 16, 1, 4),
+    LATTE(350, 75, 20, 1, 7),
+    CAPPUCCINO(200, 100, 12, 1, 6);
+
+    private final int amountOfWater;
+    private final int amountOfMilk;
+    private final int amountOfCoffeeBeans;
+    private final int amountOfDisposableCups;
+    private final int price;
```

Comment:
It is not clear which currency is used here for the `price` and what to do if the price is not an integer.

Ideally, we should use some `Money` class instead of `int`. This `Money` class could have `Currency` instance field and some other instance field(s) representing the price itself.

However, implementing a good `Money` class is not in the scope of this task.
But at least, we could rename the field to `priceInDollars` and leave a small comment that the price will always be an integer.